### PR TITLE
Add about info docs

### PR DESCRIPTION
**Pull request by mslukebo**

### CODE REVIEW

**Review by trgibeau — APPROVED** [review]

File: documentation/Using-the-SDK/Advanced/Adding-About-Information.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            CopyrightNotice = $"Copyright (C) {DateTime.Now.Year}",
+            AdditionalInformation = null,
+        };
+    }
+}
+```
+
+# Viewing Your About Information
+Where your about information appears depends on the SDK driver in which your plugin 
+is loaded. In Windows Performance Analyzer, your about information will appear as
```
[the remark this answers is not in the document]
Comment (zachnew-msft) [reply documentation/Using-the-SDK/Advanced/Adding-About-Information.md:85]: We can add it later if it makes sense.

**Review by zachnew-msft — DISMISSED** [review]

**Review by zachnew-msft — APPROVED** [review]

File: documentation/Using-the-SDK/Advanced/Adding-About-Information.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mslukebo) [reply documentation/Using-the-SDK/Advanced/Adding-About-Information.md:85]: I would personally prefer to minimize the number of PNGs being tracked by git. I'm fine with adding it if we think it's helpful, but I think it's fine without an image. Plus, that's another image to maintain should the about info window change

**Review by trgibeau — APPROVED** [review]

File: documentation/Using-the-SDK/Advanced/Adding-About-Information.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# Abstract
+
+This document outlines how to add about/author information to your plugins.
+
+# Motivation
+
+When a plugin is used by a large and/or public audience, it is useful to 
+give users an easy way to contact its owners/maintainers. SDK Drivers such as 
+WPA look for __about information__ on a `CustomDataSource` for it to
```
Comment (anramanath) [inline documentation/Using-the-SDK/Advanced/Adding-About-Information.md:9]: Did you want to update CustomDataSource in this file??

**Review by anramanath — CHANGES_REQUESTED** [review]
Are you making a sweeping change in documentation for CustomDataSource as well?

File: documentation/Using-the-SDK/Advanced/Adding-About-Information.md (hunk 2 above)
Comment (mslukebo) [reply documentation/Using-the-SDK/Advanced/Adding-About-Information.md:9]: I will update it in the other PR assuming this gets merged first.

**Review by anramanath — APPROVED** [review]

**Pull request merged**